_data/accessibility-tests/search.yml

```diff
@@ -0,0 +1,175 @@
+title: Search
+component_status: pass
+test_items:
+# General tests
+  - summary: Color is not the only method used to convey the search button's meaning.
+    summary_additional: When using the search button, "search" text or a magnifying glass icon convey the button's meaning.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.1
+  - summary: Search button background and text meet color contrast requirements. 
+    summary_additional: |
+      When you use ANDI color contrast analyzer on the search text button, the contrast between background and lettering is at least 4.5:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.3
+  - summary: Search button background and icon meet color contrast requirements.
+    summary_additional: |
+      When you use ANDI color contrast analyzer on the search icon button, the contrast between the background and icon is at least 3:1.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.7.1
+    wcag_criterion: 1.4.11
+  - summary: The width of the search button is at least 44 pixels.
+    summary_additional: |
+      When you select "inspect element" to review the button's size, you can see the total width of the button is at least 44 pixels.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 2.5.5
+  - summary: Search field and button views adjust on mobile.
+    summary_additional: |
+      When you rotate between portrait mode and landscape mode on a mobile device, the search button and field are not restricted to a single view.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.3.4
+  - summary: Page context does not change before search submission.
+    summary_additional: |
+      Updating page context or search results before expected can cause confusion for users of assistive technology. 
+      If search results are opened within a new window, advanced warning should be given alongside search 
+      instructions in the aria-label or button text.
```

Comment:
The important thing to note with this test is that it isn't restricted to screen readers but in all examples. Re-reading the criterion, it also looks like this would be acceptable if the user is advised of the behavior beforehand.

In a real-world example, this could look like a search where the results are updated in realtime on keyboard input.

What do you think of:

> Page content stays the same until you submit the search unless the user has been advised of the behavior before using the component. If search results are opened within a new window, an advanced warning should be given alongside search instructions, in the aria-label, or button text.

Alternatively switching the order of the first sentence:

> Unless notified beforehand, page content stays the same until you submit the search.

Reply:
Updated in [e590fe0](https://github.com/uswds/uswds-site/pull/2694/commits/e590fe06fb39d6330bcab1ee1f903f530c5847df). Let me know what you think @amyleadem

Reply:
Resolving this comment because the original comment has been addressed. Added a comment about the updated text in this [comment thread](https://github.com/uswds/uswds-site/pull/2694#discussion_r1636739897).